save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/table/filters/VulnerabilitiesFiltersRow.kt

```diff
@@ -80,18 +79,3 @@ val vulnerabilitiesFiltersRow: FC<NameFilterRowProps> = FC { props ->
         }
     }
 }
-
-/**
- * [Props] for filters name
- */
-external interface VulnerabilitiesFiltersProps : Props {
```

Comment:
unused, `NameFilterRowProps` used instead

Reply:
Please, do not edit `VulnerabilitiesFiltersRow`, that is what I do right now 🥺 (and yes, it will be needed)

Reply:
I will remove these changes

Reply:
Thanks

Reply:
It is still deleted, isn't it?

Reply:
<img width="826" alt="screenshot" src="https://github.com/saveourtool/save-cloud/assets/35039155/660e3c2c-232b-4958-a2e6-b4606d7ca51c">

![image](https://github.com/saveourtool/save-cloud/assets/35039155/a60edaa8-a2b6-4f02-85a0-bdf5901a3a65)